zerver/tests/test_events.py

```diff
@@ -1580,7 +1580,7 @@ def do_set_realm_property_test(self, name: str) -> None:
             invite_to_stream_policy=[3, 2, 1],
             email_address_visibility=[Realm.EMAIL_ADDRESS_VISIBILITY_ADMINS],
             bot_creation_policy=[Realm.BOT_CREATION_EVERYONE],
-            video_chat_provider=[Realm.GOOGLE_HANGOUTS, Realm.JITSI_MEET],
+            video_chat_provider=[0, 4],
```

Comment:
Let's use the declarative values for this, rather than numbers.  and I think 4 is an invalid value, right?

Reply:
We only have three available providers for now, and two of them require extra argument `domain`, `api_key` etc. I fixed the issue in separate commit.